docs/javascript-environment.md

```diff
@@ -85,24 +85,24 @@ Many standard functions are also available on all the supported JavaScript runti
 #### Browser
 
 - [CommonJS require](https://nodejs.org/docs/latest/api/modules.html)
-- [console.{log, warn, error, info, trace, table, group, groupEnd}](https://developer.chrome.com/devtools/docs/console-api)
+- [console.&#123;log, warn, error, info, trace, table, group, groupEnd}](https://developer.chrome.com/devtools/docs/console-api)
```

Comment:
Here I'd rather use a code block. `console.log` is a piece of code, so it makes sense to have the code.
Also, is a little weird that we have an HTML code for the open parenthesis and the actual closed parenthesis at the end.

Reply:
> Here I'd rather use a code block. console.log is a piece of code, so it makes sense to have the code.

I agree @cipolleschi !

At the same time, it's not real code but rather pseudo-code, not really an existing syntax but an invented one inspired by globbing patterns ([brace extensions](https://github.com/micromatch/braces))

But if we use an inline code block, this probably makes sense to be consistent on the whole section instead of just this problematic line. 

But being consistent is not easy: then appears another problem for code blocks that would eventually include multiple links: markdown links don't work inside a code block (although I have an idea to support that, cf my PR comments), and in this case, it's not possible to wrap the whole code block inside a single link. 

The problematic elements here:

![CleanShot 2023-07-31 at 11 38 19@2x](https://github.com/facebook/react-native-website/assets/749374/faeb5dd5-445a-4f35-afc1-b6b4f31248a6)

So, I thought that using an HTML char was a better solution in this context.

Another solution could be to remove the brace extension and migrate to a more verbose but real code: `Array.prototype.find` instead of `Array.prototype.{find, findIndex}`